Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -192,8 +193,10 @@ void Preset(bool _bNTSC)
 
   s_ticks_last_line_start = 0;
   s_half_line_count = 1;
-  s_half_line_of_next_si_poll = num_half_lines_for_si_poll;  // first sampling starts at vsync
+  s_half_line_of_next_si_poll = 1;  // first sampling starts at vsync
   s_current_field = FieldType::Odd;
+  s_latched_si_poll_x = 131;
```

Comment:
131 seems like an odd number, can you leave a comment what it does/is or where it came from?

Reply:
Made it a constexpr value with a comment.